### DISCUSSION TITLE
Adjust Core::transition_from_parked behavior after introducing disable_lifo_slot feature

### DISCUSSION THREAD

**Discussion by hidva** [tokio-rs/tokio#5749] [Q&A] [answered]
After introducing the `disable_lifo_slot` feature in Tokio, I think that the behavior of `Core::transition_from_parked` also needs to be adjusted. For example, when `core.run_queue.is_stealable()` is true, park should also be terminated. Additionally, to avoid task contention, `Context::park_timeout` should only wake up others when `core.run_queue.len() >= 2`.

Please let me know your thoughts on this proposed adjustment.

Best regards,
盏一

**Comment by Darksonn** [comment, marked as answer]
Please open an issue instead of a discussion. Otherwise, this can easily get lost.
